src/ScVmm/ScVmm.Autorest/examples/New-AzScVmmVM.md

```diff
@@ -0,0 +1,120 @@
+                                               "ipv4AddressType": "Dynamic",
+                                               "ipv6AddressType": "Dynamic",
+                                               "macAddressType": "Dynamic",
+                                               "nicId": "00000000-1122-0000-0001-000000000000"
+                                             }}
+OSProfileAdminPassword                     :
+OSProfileComputerName                      : ComputerName
+OSProfileOssku                             : Windows Server
+OSProfileOstype                            : Windows
+OSProfileOsversion                         : 10.0.0
+PowerState                                 : Running
+ProvisioningState                          : Succeeded
+ResourceGroupName                          : test-rg-01
+StorageProfileDisk                         : {{
+                                               "displayName": "WindowsServer.vhd",
+                                               "diskId": "00000000-1111-0000-0001-000000000000",
+                                               "diskSizeGB": 8,
+                                               "maxDiskSizeGB": 40,
+                                               "bus": 0,
+                                               "lun": 0,
+                                               "busType": "IDE",
+                                               "vhdType": "Dynamic",
+                                               "volumeType": "BootAndSystem",
+                                               "vhdFormatType": "VHD",
+                                               "createDiffDisk": "false"
+                                             }}
+SystemDataCreatedAt                        : 08-01-2024 15:05:41
+SystemDataCreatedBy                        : user@contoso.com
+SystemDataCreatedByType                    : User
+SystemDataLastModifiedAt                   : 08-01-2024 15:14:34
+SystemDataLastModifiedBy                   : 11111111-aaaa-2222-bbbb-333333333333
+SystemDataLastModifiedByType               : Application
+Type                                       : microsoft.scvmm/virtualmachineinstances
+```
+
+Enable existing SCVMM Virtual Machine in Azure
+
+### Example 2: Create new virtual machine using VM Template
+```powershell
+New-AzScVmmVM -Name "test-vm" -ResourceGroupName "test-rg-01" -VmmServerName "test-vmm" -Location 'eastus' -CloudName 'test-cloud' -TemplateName 'test-template'
+```
+
+```output
+Virtual Machine resource is returned similar to Example 1
+```
+
+
+Create new virtual machine on on-prem SCVMM
+
+### Example 3: Create new virtual machine using VM Template and customizing few properties
+```powershell
+New-AzScVmmVM -Name "test-vm" -ResourceGroupName "test-rg-01" -VmmServerName "test-vmm" -Location 'eastus' -CloudName 'test-cloud' -TemplateName 'test-template' -CpuCount 4 -AdminPassword $securePassword -Generation 2 -Tags @{"key-1"="value-1234"}
```

Comment:
Please add `$securePassword = ConvertTo-SecureString "******" -AsPlainText -Force` in Line 90